__tests__/ship.test.js

```diff
@@ -0,0 +1,50 @@
+const Ship = require("../src/ship");
+const Itinerary = require("../src/itinerary");
+
+describe("ship object", () => {
+  describe("has ports and an itinerary", () => {
+    let dover;
+    let calais;
+    let itin;
+    let ship;
+    beforeEach(() => {
+      dover = {
+        name: "Dover",
+        ships: [],
+        addShip: jest.fn(),
+        removeShip: jest.fn()
+      }
+      calais = {
+        name: "Calais",
+        ships: [],
+        addShip: jest.fn(),
+        removeShip: jest.fn()
+      }
+      itin = {ports: [dover, calais]}
+      ship = new Ship(itin);
+    });
+    test("ship can be instantiated and returns an object", () => {
+      expect(new Ship(itin)).toBeInstanceOf(Object);
```

Comment:
could use `ship` here that you've created in your `beforeEach` instead of creating a new ship, e.g.
```
expect(ship).toBeInstanceOf(Object);
```